corio.py

```diff
@@ -212,6 +214,7 @@ def log_status(parsed_input: dict, corio_start_time: datetime.time, test_failed:
         elif test_failed is None:
             status_file.write('\nTest Execution still in progress')
         else:
+            LOGGER.info(terminated_tests)
```

Comment:
Include a statement too. That will help in searching logs.

Reply:
done